pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -1263,7 +1255,7 @@ def test_parse_date_time_multi_level_column_name(all_parsers):
 )
 def test_parse_date_time(all_parsers, data, kwargs, expected):
     parser = all_parsers
-    result = parser.read_csv(StringIO(data), date_parser=conv.parse_date_time, **kwargs)
+    result = parser.read_csv(StringIO(data), **kwargs)
```

Comment:
i would leave the parsing and assert the warnings, we want to assert that the results are the same as before

Reply:
Would it be fine to replace date_parser=conv.parse_date_time with date_parser=pd.to_datetime. All the tests still pass and no warning is raised then. Plus it safes us some future changes when the functions in date_converters.py are removed from the codebase.

Reply:
so we would need to have a test that faithully replicates `conv.parse_date_time` (or hard codes the result). I am pretty sure that `pd.to_datetime` is fully replicative but we need to have tests that prove that.

Reply:
Added a pytest dateparser parameter that tests both the old function and the new function (`pd.to_datetime`). Both functions produce the same expected value, so I think we show that the new function is replicative.